CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# CONTRIBUTION
+## Codding standard
+We expect all Python code to conform to the [PEP8](https://www.python.org/dev/peps/pep-0008) standard.
+## Pull Requests(PR)
+ - Each Pull Request should focus on single repository principle.
+ - **The Pull Request should not break any of the existing functionality**
+ - <ins>Description-</ins> the PR description should explain what changes have been made, and why. In addition, they should include a link to the relevant `issue`
+ - Each PR should include 1-5 `commits`
+ - Make sure the PR passes CI checks.
+ - Each pull request requires approves of at least <ins>2 team members</ins> before merge.
```

Comment:
How about this?
 Each pull request requires approves of at least <ins>2 team members</ins> before merge.
 The team members of ResuMe are:
- Omer Shor (github user
- Yuli (github user)
- Matan  (github user)
- etc